src/users.ts

```diff
@@ -76,10 +116,26 @@ export default function(zoomApiOpts: ZoomOptions) {
       params: params
     });
   };
+  const CreateUser = function(body?: CreateUserBody) {
```

Comment:
I believe `body` should be a required parameter according to the [docs](https://marketplace.zoom.us/docs/api-reference/zoom-api/users/usercreate).

Reply:
Great catch, thanks!!